tests/unittestcoverage.py

```diff
@@ -0,0 +1,330 @@
+#!/usr/bin/env python2.7
```

Comment:
Looks good. 
However, please think about the maintenance of the duplicated scripts between the below addresses. |
* .../tests/unittestcoverage.py
* https://github.com/nnsuite/TAOS-CI/tree/master/ci/gcov